### ISSUE TITLE
Follow requests on other instances always marked as pending

### ISSUE THREAD

**Issue by joshisanonymous** [mastodon/mastodon#19525]
### Steps to reproduce the problem

1. Find a user on an instance other then your own
2. Click on that user's profile
3. Click on follow
4. Refresh the page
5. See that you're not yet following them but are instead waiting for them to approve your follow request

### Expected behaviour

Follow request should get automatically approved right away

### Actual behaviour

Follow request goes into a pending status

### Detailed description

This is happening with brand new Mastodon accounts that I know to not be set to locked, so presumably the follow request should just get automatically approved right away. The same issue was reported before, too: [https://github.com/mastodon/mastodon/issues/12431](https://github.com/mastodon/mastodon/issues/12431)

### Specifications

Mastodon 3.5.3
Firefox 106.0.2 (64 bits)
Windows 11
Instance: [mastodon.top](mastodon.top)

**Comment by ClearlyClaire** [comment]
This is most likely indicative of a federation issue (your server doesn't consider you're following a remote user before their server acks the follow request). This could be a network error, a bug on either side, or just either server struggling with the amount of work it gets. Are those follow requests still marked as pending? If so, could you share some more information, e.g. are you trying to follow a local or remote user? In case it's a remote user, which software is it using?

**Comment by joshisanonymous** [comment]
As of today, the requests are all still marked as pending. I'm on mastodon.top myself, the others are generally on mastodon.social or mstdn.social. The follow requests were all sent in the first 1 or 2 days that I joined Mastodon. However, I tried following two more users yesterday and today, respectively, and both follows worked immediately. The first was on nerdculture.de and the second on mastodon.social.

**Comment by ClearlyClaire** [comment]
Alright. It could have been a temporary fluke due to load, but that seems unlikely as it should have corrected itself by now. One reason may be the remote instance having taken moderation actions against mastodon.top, but this does not look like mastodon.social has done so, so that seems unlikely as well.

I unfortunately have no other explanation to offer at the moment, and it is difficult to investigate more without server logs.

One thing you could do is withdraw those follow requests and send them again.

**Comment by joshisanonymous** [comment]
A bit more info, today I've been able to cancel my follow requests and then successfully follow those users right after that. Additionally, I'm having a lot of trouble loading user profiles on mastodon.social or even finding them in searches as the searches often time out. I guess that instance is getting hammered a lot harder than others currently?

**Comment by ClearlyClaire** [comment]
Yes, mastodon.social is experiencing an unprecedented amount of activity and is struggling to keep up. We are working on optimizing the code and scaling the servers.

**Comment by limdingwen** [comment]
In my experience, this means that if you're trying to follow foo@bar.com, bar.com has limited (silenced) your server, and thus a follow request is required.

Silencing may happen for any reason, but for the servers I'm on, it's usually due to the server in question being too big, too noisy or under-moderated.

**Comment by ianh** [comment]
I’m running into this issue today (I’m on mastodon.social; the other accounts are on a variety of instances). It’s frustrating to have to write down every account in my notes so I can remember to come back and follow them for real. I’m sure there are users who don’t know to do this and will not realize they hadn’t followed someone they thought they had.

It would be great to see this improved at some point, even though I’m sure it will be fine once traffic dies down a bit.

**Comment by ianh** [comment]
Just wanted to say -- the pending follows did eventually go through.
